### ISSUE TITLE
Sober is not using enough VRam?

### ISSUE THREAD

**Issue by UnkwNwastaken** [vinegarhq/sober#266]
### Acknowledgement of checking existent issues

- [X] I have checked if my issue has already been reported, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

Hello, I am having problems with some experiences that has so many parts and/or has decent graphics. I've never reported a bug before. it was crashing in almost all of the maps of Midnight Racing: Tokyo, so I joined the experience again but launched Sober through the terminal. Here's the output:
```
2024-09-29T10:34:37.562Z,265.562042,776006c0,6 [LOGCHANNELS + 1] RBXCRASH: OutOfMemoryGraphics (Failed to allocate memory. size = 3979776, alignment = 1024)
RBXCRASH: OutOfMemoryGraphics (Failed to allocate memory. size = 3979776, alignment = 1024)
```
I hope this gets fixed or there is a way to fix it

**Comment by kirbix12** [comment]
It's not that it's not using enough VRAM, considering it would just keep running just fine. It's the matter of how much VRAM it can allocate at once given how much dedicated memory is loaded onto your GPU.

Low VRAM cards can tend to crash very easily and possibly fast since it can jump the gun really quickly and fill up the VRAM in the matter of seconds or minutes. This is problematic especially with NVIDIA GPUs, since their drivers are written poorly for Linux that they don't manage VRAM usage well than on Windows.

In other words: Sober crashed because your card ran out of video memory.

**Comment by UnkwNwastaken** [comment]
oh alright, thanks. I guess it's time to switch to AMD
